docs/api/paths/admin/db-_ensure_full_commit.yaml

```
@@ -8,7 +8,7 @@
 parameters:
   - $ref: ../../components/parameters.yaml#/db
 post:
-  summary: '/{db}/_ensure_full_commit'
+  summary: 'Commits any recent changes to the specified database to disk'
```

Since it doesn't really do anything maybe best to keep it plain

```suggestion
  summary: Ensure Full Commit
```